modules/sql-engine/src/integrationTest/sql/basic_queries/test_simple_table.test

```diff
@@ -0,0 +1,81 @@
+# name: test/sql/basic_queries/test_simple_table.test
+# description:  F661 Simple tables
+# group: [basic_queries]
+
+query II
+SELECT * FROM (VALUES ROW('a', 1));
```

Comment:
According to the spec, Is it necessary to use the ROW function?

Reply:
you are right, fix tests a bit